.storybook/themes/commerce.scss

```diff
@@ -1097,6 +1099,8 @@
 					grid-template-columns: repeat(12, 1fr),
 					gap: var(--global-spacing-6),
 					grid-template-rows: initial,
+					max-width: calc(var(--content-max-width) * 1px),
+					width: var(--content-scale-width),
```

Comment:
```suggestion
```
In desktop the right rail should never need this.  It should fill the entire rail width which is already taken care of by the mobile style.

Reply:
This is no longer valid with change to mobile view.